src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -752,7 +756,7 @@
     public void setSSLProtocol(String sslProtocol);
 
     /**
-     * Retrieves value of connection property 'sslProtocol'
+     * Returns the value of connection property 'sslProtocol'
```

Comment:
This change and the ones below are all missing periods at the end. Not necessarily an issue but just bringing attention to these changes to check for consistency.

Reply:
We have 'sslProtocol' here(single quotes) and "fips" (double quotes) above. Please refer to the guideline to make these consistent.